ynput/OpenPype
# Houdini: Don't error in collect review if camera is not set correctly

## Changelog Description

Do not raise an error in collector when invalid path is set as camera path. Allow camera path to not be set correctly in review instance until validation so it's nicely shown in a validation report.

## Additional info

Fixes #4869

The actual report can still be improved but would depend on [Publisher: Better error reporting](https://community.ynput.io/t/publishing-better-error-reporting/55/28) design changes in the UI too.

## Testing notes:

1. Create a review instance
2. Set an invalid camera path
3. Validation should catch it.
